source/index.html.md.erb

```diff
@@ -8064,18 +8062,15 @@ reports = response.data
 
 `GET /reports`
 
-Gets the list of all Reports that the User has access to, in alphabetical order, by name.
+Gets a summarized list of all Reports that the User has access to in alphabetical order by name. This index API does not support pagination.
```

Comment:
Experimentally, it *does*

Reply:
This one is long and convoluted.  I included my UPDATE in the description for this one: APIDOC-1060. I suspect it isn't supported because it is inconsistent?